dotnet/sample/AutoGen.BasicSamples/Program.cs

```diff
@@ -1,6 +1,4 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Program.cs
 
-using AutoGen.BasicSample;
-Console.ReadLine();
-await Example16_OpenAIChatAgent_ConnectToThirdPartyBackend.RunAsync();
+await Example07_Dynamic_GroupChat_Calculate_Fibonacci.RunAsync();
```

Comment:
Is this file typically arbitrary?

Reply:
Kinda,,, I use it to debug example,, I'll revert this change LOL